ui/page/wallets/restore_page.go

```diff
@@ -417,6 +417,19 @@ func (pg *Restore) validateSeeds() bool {
 		return false
 	}
 
+	// Compare seed with existing wallets seed. On positive match abort,
+	// import to prevent duplicate wallet. seedMatch == walled.ID if there is a match.
```

Comment:
```suggestion
	// Compare seed with existing wallets seed. On positive match abort import
	// to prevent duplicate wallet. seedMatch >= 0 if there is a match.
```

Reply:
I'm not so sure about the `seedMatch` variable name though.

Reply:
Resolved.